pytorch_lightning/loops/base.py

```diff
@@ -46,6 +47,10 @@ def __init__(self) -> None:
         self.iteration_count: int = 0
         self.trainer: Optional['pl.Trainer'] = None
 
+    @property
+    def is_connected(self) -> bool:
+        return self.trainer is not None
```

Comment:
Do we need a property for this? The check is so simple I don't think it's worth it.

Also the name is confusing as it only checks that the trainer is present, but more things could be "connected" down the line

Reply:
Removed !